spec/takeaway_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'takeaway'
```

Comment:
I like how you kept takeaway clean and used it as the "brain" to manage the other classes. In addition to checkout, I'd have still checked the methods using a double for the Order class and allowing it to receive different methods.